testing/workflows/components/workflows.libsonnet

```diff
@@ -30,12 +30,19 @@
       // outputDir is the directory to sync to GCS to contain the output for this job.
       local outputDir = testDir + "/output";
       local artifactsDir = outputDir + "/artifacts";
-      local srcDir = testDir + "/src";
-      local image = "gcr.io/mlkube-testing/kubeflow-testing";
+      // Source directory where all repos should be checked out
+      local srcRootDir = testDir + "/src";
+      // The directory containing the kubeflow/kubeflow repo
+      local srcDir = srcRootDir + "/kubeflow/kubeflow";
+      local image = "gcr.io/mlkube-testing/test-worker:latest";
```

Comment:
Same as above.

Reply:
Same as above. I think eventually we will want to start pinning things but right now its not obvious that helps and that not pinning hurts. This is an area of flux in our test infra.